portal-web/test/functional/com/liferay/portalweb/macros/RemoteAppsEntry.macro

```diff
@@ -12,21 +12,44 @@ definition {
 			value1 = "${entryHTMLName}");
 
 		Type(
-			key_id = "customElementURLs",
-			locator1 = "RemoteAppsEntry#URL",
+			locator1 = "RemoteAppsEntry#CUSTOM_ELEMENT_URL",
```

Comment:
why was the RemoteAppsEntry#URL refactored this way?

Reply:
My mistake.  The problem with the URLs was solved before my changes and it seems I didn't realize and made a mistake when comparing and merging the changes before committing them.